drv/sidecar-mainboard-controller-api/src/tofino2.rs

```diff
@@ -55,6 +55,23 @@ pub struct Status {
     error: TofinoSeqError,
     vid: Option<Tofino2Vid>,
     power: u32,
+    pcie_status: u8,
+}
+
+fn set_or_clear(p: bool) -> WriteOp {
```

Comment:
Optional cleanup: in `drv_fpga_api`, you could do

```rust
impl From<bool> for WriteOp {
    fn from(p: bool) -> Self {
        if p {
            WriteOp::BitSet
        } else {
            WriteOp::BitClear
        }
    }
}
```
 
then delete `set_or_clear` and use `.into()` below.

Reply:
Ha that's pretty neat.